python/ray/tune/execution/tune_controller.py

```diff
@@ -154,8 +154,20 @@ def _wrapped(self):
         return TrialRunnerWrapper(
             self,
             trial_executor=_FakeRayTrialExecutor(self),
-            runner_whitelist_attr={"search_alg", "get_trials", "_set_trial_status"},
-            executor_whitelist_attr={"has_resources_for_trial", "pause_trial", "save"},
+            runner_whitelist_attr={
+                "search_alg",
+                "get_trials",
+                "get_live_trials",
+                "_set_trial_status",
+                "pause_trial",
+                "stop_trial",
+            },
+            executor_whitelist_attr={
+                "has_resources_for_trial",
+                "pause_trial",
+                "save",
```

Comment:
For these methods, we can do:

```
self.get_trials.__name__
```

This way we don't ever allow/exclude a non-existent method.

Reply:
Or we can also do an assert within the wrapper class to make sure `getattr` returns something for each attr.